### ISSUE TITLE
Organize/remove excess css

### ISSUE THREAD

**Issue by spookybear0** [spookybear0/laserforce_ranking#99]
There's too much inline css that needs to be moved to a class or a separate css file. Doesn't have to be in main.css.

**Comment by chiragkhatri19** [comment]
I would love to work on this

**Comment by spookybear0** [comment]
That would be great! Let know know if you have any questions about the structure of the code.

**Comment by spookybear0** [comment]
Yes that'd be great! Just as a heads up, I might move this project to tailwind in the future, but anything contributed is heavily appreciated.
